news/2018-03-27-the-qat-gazette-march-2018.md

```diff
@@ -0,0 +1,75 @@
+
+Even if you’re not a Beatmap Nominator, don’t fear because you can still report qualified maps yourself in the [Report a Qualified Beatmap Here!](https://osu.ppy.sh/community/forums/topics/447428) thread, the same as before. The purpose of the Quality Assurance Helpers is to streamline this process and make sure all and not just some maps are checked. Everything else remains the same as before. If you have concerns over a beatmap, post your concerns and it will be looked over by a member of the Quality Assurance Team!
+
+## Within the Quality Assurance Team 
+by [Raiden](https://osu.ppy.sh/users/2239480)
+
+People join, people leave. That's how communities evolve throughout history. And the QAT is no different. 
+### New Quality Assurance Team additions
+Thanks to the QAH initiative explained above, we have determined some of our Beatmap Nominators were qualified enough to become part of our team. With the addition of the new branches explained below, each one of the new QAT members is assigned to one or various branches, making our duties more organized. 
+
+Please give a warm welcome to:
+- [-Mo-](https://osu.ppy.sh/users/2202163) and [Mir](https://osu.ppy.sh/users/8688812), who will be joining the **Disqualification** branch;
+- [Gabe](https://osu.ppy.sh/users/654108), who will partake the **Moderation** branch; 
+- and finally, [Lanturn](https://osu.ppy.sh/users/1446665), who will assist the **Metadata** branch.
+### Quality Assurance Team retirements
+Not everything can always be good news, for it is at times of loss when one really appreciates what one had
+
+- After 5 years of service and countless efforts made for the team, being both in the BAT and the QAT, the judge of mapping contests, co-creator of Beatmap Nominator tests, our long-lasting member [Irreversible](https://osu.ppy.sh/users/1287964) has made the decision of working on his own. He will still hold the Olympiad contests, though, so stay alert!
+- We also have to bid farewell to [Okoratu](https://osu.ppy.sh/users/1623405), one of our former QAT leaders. He has chosen to follow his own path while continuing to contribute to the mapping and modding scene by remaining a **Beatmap Nominator**. Being one of the pioneers of the **United Beat-Knights of the Ranking Criteria**, he worked tremendously hard to make our Ranking Criterias a dynamic set of rules that adapt to the evolution of the mapping scene. On top of that, he is also one of the founders and main organizers of the **Mentorship Program** as well as one of the promoters of **Mapping with Rewards**, making his departure indubitably a great loss for the QAT.
+### Overview of the new workfields (Branches) 
```

Comment:
Probably best to put this first, since you mention branches in the new additions section and it doesn't make sense until you explain it here. Also the branching system kinda explains "hey this is why we needed more people" and leads into that nicely.

Might be nice to explain that (L) is for "Leader" unless I missed that somewhere.

Reply:
okay